core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -4146,9 +4149,52 @@ private static List<Expression> args(RexCall call,
         if (argValueList.size() == 2) {
           return list.append(argValueList.get(1));
         }
+        if (argValueList.size() == 1) {
+          return list.append(Expressions.constant(Math.exp(1)));
+        }
+        // fall through
+      case "LN":
+        return list.append(Expressions.constant(Math.exp(1)));
+      case "LOG10":
+        return list.append(Expressions.constant(BigDecimal.TEN));
+      default:
+        throw new AssertionError("Operator not found: " + call.getOperator());
+      }
+    }
+  }
+
+  /** Implementor for the {@code LN}, {@code LOG}, and {@code LOG10} operators.
```

Comment:
This javadoc looks identical to the one above. If the comment can be the same there's a chance we probably don't need two different implementors.

Would you mind either: a) making the comment more clear so we know why this implementor is needed or b) putting it all in the original LogImplementor?